samples/wifi/twt/modules/traffic_gen/Kconfig

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+#
+config TRAFFIC_GEN
+	bool "Traffic generation module"
+	select NET_CONTEXT_RCVTIMEO
+
+choice TRAFFIC_GEN_SELECT_ROLE
+	prompt "APP should be either client or server"
+	default TRAFFIC_GEN_CLIENT
+
+config TRAFFIC_GEN_CLIENT
+	bool "Traffic gen module acts as a Client"
+	help
+	  Send uplink traffic to traffic gen server
+
+config TRAFFIC_GEN_SERVER
+	bool "Traffic gen module acts as a Server"
+	help
+	  Receive downlink traffic from traffic gen client (A python
+	  client running in Cloud).
+
+endchoice
+
+config TRAFFIC_GEN_DURATION
```

Comment:
Can we write all Kconfig Help in a similar way to make it consistent with other Kconfigs?